Makefile

```diff
@@ -180,21 +180,38 @@ DEMO_TARGETS = bookbuyer bookthief bookstore bookwarehouse tcp-echo-server tcp-c
 DEMO_BUILD_TARGETS = $(addprefix build-, $(DEMO_TARGETS))
 .PHONY: $(DEMO_BUILD_TARGETS)
 $(DEMO_BUILD_TARGETS): NAME=$(@:build-%=%)
+ifeq ($(OS), windows)
+	EXT=.exe
+endif
 $(DEMO_BUILD_TARGETS):
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ./demo/bin/$(NAME)/$(NAME) ./demo/cmd/$(NAME)
+	GOOS=$(OS) GOARCH=amd64 CGO_ENABLED=0 go build -o ./demo/bin/$(NAME)/$(NAME)$(EXT) ./demo/cmd/$(NAME)
 	@if [ -f demo/$(NAME).html.template ]; then cp demo/$(NAME).html.template demo/bin/$(NAME); fi
 
 .PHONY: demo-build
-demo-build: $(DEMO_BUILD_TARGETS) build-init-osm-controller build-osm-controller build-osm-injector build-osm-crds build-osm-crd-converter
+demo-build:
+	make OS=linux $(DEMO_BUILD_TARGETS) build-init-osm-controller build-osm-controller build-osm-injector build-osm-crds build-osm-crd-converter
+
+.PHONY: demo-build-windows
+demo-build-windows:
+	make OS=windows $(DEMO_BUILD_TARGETS) build-init-osm-controller build-osm-controller build-osm-injector build-osm-crds build-osm-crd-converter
```

Comment:
I'd be in favor of removing these altogether since I don't think they're dependencies for anything else. If anyone actually uses these then I'm fine keeping them, but anywhere we can remove a big list of images is a plus in my book.

Reply:
You mean delete the make rules `demo-build` and `demo-build-windows`

Reply:
Yes